server/server.go

```diff
@@ -2826,6 +2826,7 @@ func (s *Server) StartProfiler() {
 		Addr:           hp,
 		Handler:        http.DefaultServeMux,
 		MaxHeaderBytes: 1 << 20,
+		ReadTimeout:    time.Second * 5,
```

Comment:
5 seconds maybe too low for the profile port? Bump to 15 seconds maybe?

Reply:
It's only for reading the request body, which isn't really used on the profiling port. I checked that pulling long-running profiles works and it's fine.